client/components/AuthForm-styled.js

```diff
@@ -0,0 +1,117 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import { authenticate } from '../store';
+import Form from 'react-bootstrap/Form';
+import Button from 'react-bootstrap/Button';
+import { useNavigate } from 'react-router-dom';
+import { useHistory } from 'react-router-dom';
+
+/**
+ * COMPONENT
+ */
+const AuthForm = (props) => {
+  const [showSignup, setShowSignup] = React.useState(false);
+  const { name, displayName, handleSubmit, error } = props;
+
+  const history = useHistory();
+
+  const onSubmitClick = () => {
+    history.push('/signup');
+    setShowSignup(true);
```

Comment:
I'm not sure you want to be using a piece of local state to manage this. It would probably be better to either leverage react router to determine whether you're on the signup or login pages, or pass some kind of prop to the component